python_modules/libraries/dagster-looker/dagster_looker_tests/api/test_build_defs.py

```diff
@@ -81,7 +82,7 @@ def test_build_defs(
     assert len(asset_specs_by_key) == 3
 
     expected_lookml_view_asset_key = AssetKey(["view", "my_view"])
-    expected_lookml_explore_asset_key = AssetKey(["my_model::my_explore"])
+    expected_lookml_explore_asset_key = AssetKey(["my_model_my_explore"])
```

Comment:
This is technically a breaking change, right now this name is invalid as an `input` so we can't put it upstream of an executable asset